docs/kyma/docs/04-05-aks-installation.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Install Kyma on an AKS cluster
+type: Installation
+---
+
+This Installation guide shows developers how to quickly deploy Kyma on an [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster. Kyma installs on a cluster using a proprietary installer based on a Kubernetes operator.
+
+## Prerequisites
+- A domain for your AKS cluster
+- [Microsoft Azure](https://azure.microsoft.com)
+- [Docker](https://www.docker.com/)
+- [Docker Hub](https://hub.docker.com/) account
+- [az](https://docs.microsoft.com/pl-pl/cli/azure/install-azure-cli)
+- set the environment variables
+
+### Environment variables
+
+Set the following environment variables:
+```
+export RS_GROUP={YOUR_RESOURCE_GROUP_NAME}
+export CLUSTER_NAME={YOUR_CLUSTER_NAME}
+export REGION={YOUR_REGION} #westeurope
+```
+
+If you use a custom domain, set also these variables:
+```
+export DNS_DOMAIN={YOUR_DOMAIN} # example.com
+export SUB_DOMAIN={YOUR_SUBDOMAIN} # cluster (in this case the full name of your cluster is cluster.example.com)
+```
+
+Create a resource group that will contain all your resources:
+```
+az group create --name $RS_GROUP --location $REGION
+```
+
+>**NOTE:** If you don't own a domain which you can use or you don't want to assign a domain to a cluster, see the **Install Kyma on an AKS cluster with wildcard DNS** document which shows you how to create a cluster-based playground environment using a wildcard DNS provided by xip.io. 
```

Comment:
Maybe we can link to this document as the linking option is already enabled on UI?